lib/filesystem/CCompressedStream.cpp

```diff
@@ -26,7 +26,7 @@ si64 CBufferedStream::read(ui8 * data, si64 size)
 {
 	ensureSize(position + size);
 
-	auto start = buffer.data() + position;
+	auto *start = buffer.data() + position;
```

Comment:
```suggestion
	auto * start = buffer.data() + position;
```

Reply:
all formatting is passed via clang-format.